docker/docker-chaos/metrictank.ini

```diff
@@ -457,7 +457,7 @@ connection-check-timeout = 30s
 [memory-idx]
 enabled = false
 # enables/disables querying based on tags
-tag-support = false
+tag-support = true
 # enables/disables querying based on meta tags
 meta-tag-support = false
```

Comment:
does it not make sense to enable meta tags by default either?

Reply:
not sure, at least while bigtable isn't supported i think we shouldn't enable it by default. otherwise the api endpoints on the BT instances might look like everything should work but the meta records won't get persisted.

Reply:
implementing that on BT is an issue of this sprint btw, trying to get that done this week

Reply:
We don't completely understand all of the performance implications of enabling meta tags by default yet. I think we should wait a few months.